vendor/assets/javascripts/changed_form_modal.js

```diff
@@ -0,0 +1,18 @@
+var NR = Cmsify.NestedResource.prototype;
+var CFM = Cmsify.ChangedFormModal = function(el) {
+  this.$el = $(el);
+  this.form = this.$el.find('form');
+  this.modal = UIkit.modal(this.$el.find('#js-unsaved-modal'));
```

Comment:
Line 5 of this: https://github.com/plyinteractive/spr-admin/pull/88/files#diff-319d6fb7182fce1d33bc458c66fa792e it's the modal that pops up when you have unsaved changes as per this design: https://projects.invisionapp.com/d/main#/console/9694214/216096470/preview I can switch it over to a class.